release_dashboard/lib/services/local_conductor.dart

```diff
@@ -32,8 +32,11 @@ class LocalConductorService extends ConductorService {
   Directory get rootDirectory => fs.directory(platform.environment['HOME']);
   File get stateFile => fs.file(defaultStateFilePath(platform));
 
-  static const String frameworkUpstream = 'https://github.com/flutter/flutter';
-  static const String engineUpstream = 'https://github.com/flutter/engine';
+  @override
+  late String engineCheckoutDirectory;
```

Comment:
These should be getters to prevent overriding the data outside of the class

Reply:
I agree you should make this a getter, but I think you should also consider how to handle this getting called before it's been initialized.